libswirl/hw/sh4/dyna/driver.cpp

```diff
@@ -297,7 +297,7 @@ u32 DYNACALL rdv_DoInterrupts_pc(u32 pc) {
 	//We can only safely relocate/etc stuff here, as in other generic update cases
 	//There's a RET, meaning the code can't move around
 	//Interrupts happen at least 50 times/second, so its not a problem ..
-	if (rebuild_counter == 0)
+	//if (rebuild_counter == 0)
```

Comment:
Accidental changes?

Reply:
I think that's a merge derp, but yeah removed.